src/test/acceptance/features/address-lookup.feature

```diff
@@ -20,4 +20,5 @@ Feature: Select address
       When I enter a postcode
       Then I am shown the select address page
       And I am shown a list of addresses
+      And I am shown an address not listed link that navigates to the manual address page
```

Comment:
Does this step need `that navigates to the manual address page`? That should be assertion but I think its superfluous to the step itself.

Reply:
I'm not sure I understand. Are you saying the assertion isn't needed?

Reply:
I think Mike' saying that the name of the condition could simply be: `And I am shown an address not listed link`. The assertion should remain the same under the covers

Reply:
I see, updated the definition to `I am shown an address not listed link`